draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -612,14 +620,16 @@ Upon reception of the MP_FAST_CLOSE and validation of the Key Data at the peer h
 
 
 ~~~~
-  +--------+--------+--------+-----------+-------------+
-  |00101110| Length |00000011|Key Type(1)| Key Data(1) | ->
-  +--------+--------+--------+-----------+-------------+
+                        1                    2                    3     
+   0 1 2 3 4 5 6 7  8 9 0 1 2 3 4 5  6 7 8 9 0 1 2 3  4 5 6 7 8 9 0 1
+  +----------------+----------------+----------------+----------------+------------------+
+  |0 0 1 0 1 1 1 0 |     Length     |0 0 0 0 0 0 1 1 |  Key Type (1)  |  Key Data (2)  ... ->
+  +----------------+----------------+----------------+----------------+------------------+
```

Comment:
Introducing a blank between each character exceeds the line length, see:
https://xml2rfc.tools.ietf.org/cgi-bin/xml2rfc-dev.cgi?url=https://raw.githubusercontent.com/shjohnson-bt/ietf-multipath-dccp/master/draft-ietf-tsvwg-multipath-dccp.mkd&inputtype=kramdown&format=v3ascii

Only 32 bit header sizes can be kept within page width when blank is used.